ideas/README.md

```diff
@@ -12,7 +26,7 @@ Or from one of the following sources:
 * [sleepyfox London Code Dojo](https://github.com/sleepyfox?utf8=%E2%9C%93&tab=repositories&q=dojo)
 * [Code Kata](http://codekata.com/)
 
-## Repeat Code Katas
+### Repeat Code Katas
 
 It goes without saying, but code katas are meant to be repeated.
```

Comment:
Maybe suggest reviewing your previous Katas? Is there anything you could change to make your code more SOLID, more readable, can you get a peer review of something you wrote?

Reply:
Appreciating some of this is touched on in the examples